.github/workflows/prune_workflow.yml

```diff
@@ -0,0 +1,60 @@
+name: Prune workflow runs
+on:
+  schedule:
+    # Run monthly, at 00:00 on the 1st day of month.
+    - cron: '0 0 1 * *'
```

Comment:
daily?

Reply:
Daily seems too aggressive as these runs would be consulted for PRs being reviewed. A month seems like a fair start for when to perform pruning. If we need to prune sooner, we run manually.